webapp/src/main/java/org/opentestsystem/rdw/admin/service/impl/DefaultSchoolService.java

```diff
@@ -24,7 +24,7 @@ public DefaultSchoolService(@NotNull final SchoolRepository repository) {
 
     @Override
     public Set<Organization> getSchools(@NotNull final User user) {
-        final Permission permission = user.getPermissionsById().get("INDIVIDUAL_PII_READ");
+        final Permission permission = user.getPermissionsById().get("GROUP_READ");
```

Comment:
I don't think this is used anywhere, and was just a direct copy / paste from Reporting?  We don't care about PII permissions here though, so I'm repurposing for GROUP_READ.